src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp

```diff
@@ -576,310 +335,89 @@ namespace
                 return FALSE;
             }
         }
-    };
-
-    // ------------------------------------------------------------------------------------------------
-    // Output panel.
-    // ------------------------------------------------------------------------------------------------
-
-    struct OutputPanel
-      : public PanelBase
-    {
-        IRendParams*            m_rend_params;
-        RendererSettings&       m_settings;
-        HWND                    m_rollup;
-        HWND                    m_radio_render_only;
-        HWND                    m_radio_save_only;
-        HWND                    m_radio_save_and_render;
-        HWND                    m_static_project_filepath;
-        ICustEdit*              m_text_project_filepath;
-        ICustButton*            m_button_browse;
-        ICustEdit*              m_text_scale_multiplier;
-        ISpinnerControl*        m_spinner_scale_multiplier;
-
-        OutputPanel(
-            IRendParams*        rend_params,
-            RendererSettings&   settings)
-          : m_rend_params(rend_params)
-          , m_settings(settings)
-        {
-            m_rollup =
-                rend_params->AddRollupPage(
-                    g_module,
-                    MAKEINTRESOURCE(IDD_FORMVIEW_RENDERERPARAMS_OUTPUT),
-                    &dialog_proc_entry,
-                    L"Output",
-                    reinterpret_cast<LPARAM>(this));
-        }
-
-        ~OutputPanel() override
-        {
-            ReleaseISpinner(m_spinner_scale_multiplier);
-            ReleaseICustEdit(m_text_scale_multiplier);
-            ReleaseICustButton(m_button_browse);
-            ReleaseICustEdit(m_text_project_filepath);
-            m_rend_params->DeleteRollupPage(m_rollup);
-        }
 
-        void init(HWND hwnd) override
+        void init(HWND hwnd)
```

Comment:
No longer overriding a base class method?

Reply:
Yes, this class does not inherit PanelBase. Actualy the only PanelBase class is AboutPanel. 
Output and System rollouts just use extended ParamMap2UserDlgProc classes.